app/controllers/videos_controller.rb

```diff
@@ -0,0 +1,25 @@
+class VideosController < ApplicationController
+  def new
+    @video = Video.new
+  end
+
+  def create
+    @video = current_streamer.videos.build(video_params)
+    if @video.save
+      redirect_to video_path(@video.id)
+      flash[:notice] = 'Video cadastrado com sucesso!'
```

Comment:
```suggestion
      redirect_to @video, notice: 'Video cadastrado com sucesso!'
```

Da pra simplificar um pouco essa linha usando algumas funções legais do Rails. O que acha de usar I18n?

Reply:
feito